clients/ui/bff/internal/integrations/k8s.go

```diff
@@ -203,14 +203,20 @@ func (kc *KubernetesClient) GetServiceDetails() ([]ServiceDetails, error) {
 				continue
 			}
 
-			displayName := service.Annotations["displayName"]
+			displayName := ""
+			description := ""
+
+			if service.Annotations != nil {
+				displayName = service.Annotations["displayName"]
```

Comment:
Would this cause a key error if you had annotations but not `displayName` or `description` in there?

Reply:
Nop! If the key doesn't exist, it will return an empty string ("").